ipython_parallel/_version.py

```diff
@@ -1,2 +1,9 @@
-version_info = (4, 0, 0, 'dev')
+_version_major = 4
+_version_minor = 0
+_version_patch = 0
+_version_extra = 'dev'
```

Comment:
The conf script for ipython uses `_version_extra` to insert a disclaimer:
https://github.com/ipython/ipython/blob/master/docs/source/conf.py#L72

I wanted to also support this, should I take it out?